main/tests/test_circle.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+
+class TestCircle:
+
+    def test_get_coords(self):
+        circle = Circle()

```

Comment:
To be able to get coords, you have to initialize a value in the constructor or set the coords with the method. My recommendation is to initialize it with the constructor, so this test isn't dependent on the set_coords method working.

Reply:
got it, I will put in values for the constructor to initialize with